### PR TITLE
Meeting

### PR DESCRIPTION
**Pull request by HeloiseFerla**

### CODE REVIEW

**Review by BBriset — COMMENTED** [review]
Dont think we need that "tatus" files, otherwise great job!

**Review by BBriset — DISMISSED** [review]
good according to me

**Review by BBriset — DISMISSED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -20,26 +20,25 @@ function App() {
...
+          element={<Home setTheme={setTheme} theme={theme} />}
+        />
+        <Route path="EazyCard" element={<EazyCard />} />
+        <Route path="Food" element={<Food />} />
+        <Route path="Office" element={<Office />} />
+        <Route path="meetingRooms" element={<Meeting />} />
+        <Route path="meetingRooms/:id" element={<Room />} />
+        <Route path="Supplies" element={<Supplies />} />
+        <Route path="LostCard" element={<LostCard />} />
+        <Route path="ProfilPage" element={<ProfilPage />} />
```
Comment (LoicBrassart) [inline src/App.jsx]: Be consistent about your routes' case (upper vs lower)

File: src/pages/Meeting/Room/Share/SharePopup/style.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        background: #ececec;
+        margin: 15px;
+        padding: 5px;
+        border-radius: 5px;
+        -webkit-box-shadow: 0px 7px 12px -3px #afafaf;
+        box-shadow: 0px 7px 12px -3px #afafaf;
+      }
+    }
+    }
+  }
```
Comment (LoicBrassart) [inline src/pages/Meeting/Room/Share/SharePopup/style.jsx:36]: Indentation problem. Isn't Prettier installed ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
I'm almost you don't need all those `z-index`

**Review by BBriset — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**